src/publisher/Init.ts

```diff
@@ -33,6 +33,11 @@ export function initPublisher(
     completionHandler
   );
 
+  const audioSource =
+    properties?.audioSource === null ? undefined : properties?.audioSource;
```

Comment:
Just for my own future knowledge, I guess this will break if we set the source to `null`, and that's why we need to make it explicitly `undefined` instead, right?

Reply:
@lazeratops this is because Opentok's types allow you to pass in `null`, however, Daily's `startCamera` does _not_ allow you to pass in `null`. If this wasn't a shim project, I'd change the API itself... But because we need to support their API I had to do this check.